lib/language.js

```diff
@@ -54,10 +54,14 @@ exports.errors = {
         length: 'must be {{limit}} bytes'
     },
     date: {
-        base: 'must be a number of milliseconds or valid date string',
+        base: 'must be a valid date string or timestamp',
```

Comment:
Since the javascript mode is essentially keeping the behavior we already had, let's keep the errors identical.